ros_connect.bash

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+echo "Be sure to call this script: source ./ros_connect {ROS_MASTER_IP} or source ./ros_connect {ROS_MASTER_IP} {ROS_TALKER_IP}"
```

Comment:
Maybe we can change this a little to express that the hostname instead of the IP, if everything is correctly set up, will also work.

Reply:
Do you mean just change the echo ? Or something else in the code?

Reply:
Yes, only change the echo message.